docs/tutorials/crates/Cargo.lock

```diff
@@ -29,52 +29,30 @@ dependencies = [
 ]
 
 [[package]]
-name = "anstream"
```

Comment:
The changes in this file are a bit larger than expected but I think it's because this lockfile was already stale since we removed ansream